documentation/modules/metrics/proc_metrics-deploying-prometheus-operator.adoc

```diff
@@ -12,17 +12,20 @@ To deploy the Prometheus Operator to your Kafka cluster, apply the YAML bundle r
 
 . Download the `bundle.yaml` resources file from the repository.
 +
+[source,shell,subs="+quotes,attributes+"]
+curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml > prometheus-operator-deployment.yaml
++
+** Update the namespace by replacing the example `my-namespace` with your own
++
 On Linux, use:
 +
 [source,shell,subs="+quotes,attributes+"]
-curl -s https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml | sed -e '/[[:space:]]\*namespace: [a-zA-Z0-9-]*$/s/namespace:[[:space:]]\*[a-zA-Z0-9-]*$/namespace: _my-namespace_/' > prometheus-operator-deployment.yaml
```

Comment:
![image](https://github.com/strimzi/strimzi-kafka-operator/assets/43372967/f5d881bf-727e-45bc-aa20-938b6b5010db)

Problems with current doc:
1. In the download command, we include the `sed` command to update the namespace name with `my-namespace`, but later, we ask users to replace the example `namespace` with your own, which is wrong. Updated with: (a) download the yaml (b) update namespace (c) apply yaml
2. The provided `sed` command for macOS cannot work